src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -90,13 +90,22 @@ public void ConfigureServices(IServiceCollection services)
 
 #### List of metrics produced
 
-The instrumentation is implemented based on [metrics semantic
-conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpserverduration).
-Currently, the instrumentation supports the following metric.
+The instrumentation is implemented based on
+[metrics semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/4bbb8c907402caa90bc077214e8a2c78807c1ab9/docs/http/http-metrics.md).
 
-| Name  | Instrument Type | Unit | Description |
-|-------|-----------------|------|-------------|
-| `http.server.duration` | Histogram | `ms` | Measures the duration of inbound HTTP requests. |
+Have you opt-ed into the new Http Semantic Conventions using `OTEL_SEMCONV_STABILITY_OPT_IN`?
+
+* If yes, the instrumentation supports the following metric.
+
+    | Name  | Instrument Type | Unit | Description |
+    |-------|-----------------|------|-------------|
+    | `http.server.request.duration` | Histogram | `s` | Measures the duration of inbound HTTP requests. |
```

Comment:
it'd be good to callout here that SDK uses a special treatment for this metric by using a suitable buckets.
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4820/files

Reply:
How much information are you looking for?
Is it sufficient to say "http.server.request.duration uses unique buckets suitable for seconds" ?

I could include a link to the spec which shows the buckets:
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration

Reply:
something along the lines of
"This metric is emitted in "s" as unit, as per the semantic convention guide. The convention also recommends to provide a custom buckets (link), but that capability is not yet available in OTel .NET. A workaround is already included in the OTel SDK (link), which specially treats this metric, and uses the recommended buckets"

^ feel free to edit/modify!

Reply:
updated. please re-review

Reply:
I'd highlight/make it clear that the workaround is "OpenTelemetry SDK specially treats this metric and uses a diff. default for histogram buckets, matching the recommended buckets" /similar wording. This will ensure most folks do not have any need to go and read the linked PR.

Reply:
the linked PR is very light on details. That must be fixed separately. Specifically, 
1. it should list the exact name of the metrics for which SDK uses a diff. default.
2. make it clear that the special treatment is only for the default case. If user overrides the buckets, then that'll always be respected. 

(Not part of this PR, but this is essential to get the whole transition smoother for end users)